mix.exs

```diff
@@ -9,7 +9,7 @@ defmodule BroadwayCloudPubSub.MixProject do
     [
       app: :broadway_cloud_pub_sub,
       version: @version,
-      elixir: "~> 1.11",
+      elixir: "~> 1.12",
```

Comment:
Bump to 1.12 where the security support just ended recently, see https://endoflife.date/elixir

Not sure if we should sync with Broadway min Elixir requirement, which is still set to 1.7, see https://github.com/dashbitco/broadway/blob/9eaf2140b6c1f36b25f990a7c74052bece26aadc/mix.exs#L11

Reply:
It is end of life but we should not be the ones forcing users to update. If we can still support 1.11, then it should. :)

Reply:
Noted, and updated.